rest-client/src/main/java/org/sagebionetworks/bridge/rest/ErrorResponseInterceptor.java

```diff
@@ -72,41 +74,46 @@ private void throwExceptionOnErrorStatus(String url, int statusCode, JsonElement
                 message = "There has been an error on the server";
             }
         }
-
-        BridgeSDKException e;
-        if (statusCode == 401) {
-            e = new NotAuthenticatedException(message, url);
-        } else if (statusCode == 403) {
-            e = new UnauthorizedException(message, url);
-        } else if (statusCode == 404) {
-            e = new EntityNotFoundException(message, url);
-        } else if (statusCode == 410) {
-            e = new UnsupportedVersionException(message, url);
-        } else if (statusCode == 412) {
```

Comment:
unknownErrorInvalidBodyNoMessage() verifies an HTML response with a status code is correctly converted to BridgeSDKException